.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,21 +8,21 @@
 
 ## Functional Changes
 <!-- Put an X next any that apply -->
-[ ] New map or map update
-[ ] New Feature
-[ ] Feature update or enhancement
-[ ] Feature Removal
-[ ] Code Cleanup or refactor
-[ ] Configuration Change
-[ ] Bug fix:  <!-- Link to bug issue or forum post here -->
-[ ] Other:   <!-- Please specify -->
+[] New map or map update
+[] New Feature
+[] Feature update or enhancement
+[] Feature Removal
+[] Code Cleanup or refactor
+[] Configuration Change
+[] Bug fix:  <!-- Link to bug issue or forum post here -->
+[] Other:   <!-- Please specify -->
```

Comment:
Well using monospace fonts this makes a difference, because with this change the boxes no longer have the same width once you place an x in one of them, but ok

Reply:
yeah, small hesitation there, but it's okay, the 'wide' checkbox should draw more attention to what is checked. Meanwhile we avoid having to do that annoying delete of the space :grin: